### ISSUE TITLE
Categorizing Matched, Categorizing Unmatched & 3 Categorizing Unmatched

### ISSUE THREAD

**Issue by XiaomeiSchnittgrund** [danb-dliflc/GLOSSActivities#42]
**Templates Name**: Categorizing Matched
**Lesson tested:** cm_geo306, cm_geo401, cm_soc086, fr_cul303, hbw_pol302, hbw_tec303, ps_sci327, ps_soc345 and ru_geo005, 
**Templates Name:** Categorizing Unmatched_
**Lesson tested:**  cm_env070, fr_sci308, fr_cul313, hbw_soc310, hbw_soc418, ps_cul415, ps_ecn405, ru_soc242 and ru_tec001
**Templates Name**:  3 Categorizing Unmatched
**Lesson tested:** Chinese N/A, fr_cul307, fr_geo426, hbw_sci310, hbw_soc324, ps_cul333, ps_cul341, ru_geo022 and ru_soc319

1)	All scroll bars are still hard to move in IE 11, but there is no problem in the others.

2)	It seems that each column has more (space) boxes than is needed.  
3)	It seems that all of instructions may have to change a word: instead of “put” should be “**drag**’” the words /phrases into the appreciated category.  
4)	**GLOSSAY**: (Remains no change from last time I commented on) TL should be in the first column, not English; the layout of vocabulary print out list still remains one column instead of two. Fonts are a bit too light colored. 

**Note:** Egyptian  N/A

**Comment by wendy-dliflc** [comment]
1.) We decided that we are leaving the scroll bars.
2.) Discussed with Julia, there is always one more spot than the number of answers. 
3.) This has to do with the text that's in the instructions field (what was put in the generator), not code related; but, I discussed it with Julia and we are leaving the wording. I also think that the word "put" still makes sense even though it's not as specific as the word "drag." 
4.) The font color is what we were given for the design color. Also checked with Julia about this and we are using the color that is there. 

Will add into Glossary issues the following: 
  a.) TL should be in the first column, not English
  b.) The layout of vocabulary print out list still remains one column instead of two
